### ISSUE TITLE
Accessibility: Progress component is not announced by screen reader as progress bar

### ISSUE THREAD

**Issue by kpustakhod** [ant-design/ant-design#38444]
### Reproduction link

[https://ant.design/components/progress/#components-progress-demo-line](https://ant.design/components/progress/#components-progress-demo-line)

### Steps to reproduce

Turn on screen reader (VoiceOver or NVDA), try to navigate with arrows to propress. 

### What is expected?

Screen reader should inform user that this is a progress bar.

### What is actually happening?

Voice Over reads out "30%, group", "30%", "end of 30%, group" that is not informative.
NVDA:
<img width="1307" alt="Screenshot 2022-11-08 at 12 27 40" src="https://user-images.githubusercontent.com/16884481/200572912-35c4e270-749e-4ef4-b091-84a47a652381.png">

| Environment | Info |
| --- | --- |
| antd | 4.24.1 |
| React | 17 |
| System | MacOS |
| Browser | Chrome |

---
Need to add role="progressbar" to component wrapper.
